enum.go

```diff
@@ -130,28 +268,6 @@ func AllConnectiveEnum() []string {
 	}
 }
 
-// ContactType represents the method of contact.
-type ServiceOwnershipCheckContactType string
-
-const (
-	ServiceOwnershipCheckContactTypeAny         ServiceOwnershipCheckContactType = "any"          // Any contact method.
```

Comment:
I think this was put in here because the service ownership check does not actually take an enum but there are only certain valid values.  And it also doesn't follow just the `ContactType` enum because "any" is not a valid `ContactType` but it is allowed in the `ServiceOwnershipCheck`

Seems like something that would need to be fixed on the Ruby side (using an Enum in that check type) before we can remove it here.

CC @maxim-opslevel to double check my understanding